tox.ini

```diff
@@ -191,6 +191,7 @@ commands =
 [testenv:treon]
 deps =
     treon
+    seaborn # this should be part of plotting?
```

Comment:
Yeah it should be part of plotting… I wonder why it isn’t

Reply:
now it seems to be fine